packages/ember-metal/lib/property_get.js

```diff
@@ -5,12 +5,6 @@
 import { assert } from 'ember-debug';
 import { isPath } from './path_cache';
 
-const ALLOWABLE_TYPES = {
```

Comment:
Why is this removed?

Reply:
why to have it :P, even if other types passed `get` will return `undefined`

Reply:
Generally speaking, its best to review the git history to understand a given bit of code's purpose.

* https://github.com/emberjs/ember.js/issues/13444
* https://github.com/emberjs/ember.js/pull/13449
* https://github.com/emberjs/ember.js/pull/13461

Specifically answering your question though is https://github.com/emberjs/ember.js/pull/13449#r62256688.

Reply:
added test for those cases if something missing will add it, I think all cases handled correctly